### ISSUE TITLE
[Roadmap] Torchbench V3 release

### ISSUE THREAD

**Issue by xuzhao9** [pytorch/benchmark#1519]
We are working towards releasing TorchBench V3 by Apr 10, 2023.

The V3 nightly CI will be A100 GPU-only. This is because:
1. running all models CPU tests is too slow and takes over 24 hours to finish.
2. the Intel team is working on a new 'CPU' userbenchmark to track CPU performance
3. Before 2) is done, we will keep running V2 CI to track both CPU and GPU performance.

To release the V3 nightly CI, we need to do the following:

# Milestone 1
- [x] Run the model stableness test (Done, see https://github.com/pytorch/benchmark/actions/runs/4585968352)
- [x] Run the V3 workflow without score (https://github.com/pytorch/benchmark/pull/1523)
- [x] Add the V3 score metric to the workflow (https://github.com/pytorch/benchmark/pull/1538)
- [x] Build the internal dashboard to track the score and regression

# Milestone 2
- [x] Use userbenchmark regression detector to auto-detect regressions
- [x] Use userbenchmark bisector to bisect regressions

**Comment by xuzhao9** [comment]
The model stableness workflow is done: https://github.com/pytorch/benchmark/actions/runs/4585968352.
Run each model test for 15 iterations. 16 tests are unstable, 141 tests are stable.

List of unstable tests:
- [ ] LearningToPaint train
- [ ] timm_efficientdet eval
- [ ] soft_actor_critic train
- [ ] resnext50_32x4d train
- [ ] lennard_jones eval
- [ ] functorch_maml_omniglot eval
- [ ] functorch_dp_cifar10 train
- [ ] drq eval
- [ ] detectron2_maskrcnn_r_50_fpn train
- [ ] detectron2_maskrcnn_r_50_c4 eval
- [ ] detectron2_fcos_r_50_fpn eval
- [ ] detectron2_fasterrcnn_r_50_c4 eval
- [ ] detectron2_fasterrcnn_r_50_c4 train
- [ ] detectron2_fasterrcnn_r_101_fpn eval
- [ ] detectron2_fasterrcnn_r_101_c4 eval
- [ ]  densenet121 eval
- [ ] LearningToPaint train
